src/pfe/file-watcher/server/test/functional-test/lib/utils.ts

```diff
@@ -164,3 +164,27 @@ export async function rebuildProject(socket: SocketIO, projData: ProjectCreation
         fail(`failed to find ${targetEvent} for rebuild project ${projData.projectType}`);
     }
 }
+
+export async function removeProjectFromRunningBuild(projData: ProjectCreation): Promise<void> {
+    if (project_configs.needManualReset[projData.projectType]["runningBuildQueue"]) {
+        const runningBuildQueue = await removeProjectFromRunningBuildQueue(projData.projectID);
+        expect(runningBuildQueue);
+        expect(runningBuildQueue).to.not.include(projData.projectID);
+    }
+}
+
+export async function setBuildStatus(projData: ProjectCreation, status?: string): Promise<void> {
```

Comment:
probably a good idea to add a comment here as well why're manually setting build status from the tests

Reply:
Done.